_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -76052,7 +76069,7 @@
 	input_tag = "n2o_in";
 	name = "Nitrous Oxide Supply Control";
 	output_tag = "n2o_out";
-	sensors = list("n2o_sensor"="Tank")
```

Comment:
This needs to be reverted, the other maps are in this format and meta should not be any different.